bundle/config/mutator/resourcemutator/apply_target_mode_test.go

```diff
@@ -27,6 +25,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// FIXME apply_target_mode_test should be untied from apply_presets_test
+// and apply_target_mode with validate_target_mode should be moved into mutators package.
```

Comment:
Question - why should they be moved?

Reply:
Because `validate_target_mode` isn't only applied to resources. For example, it validates other sections of bundle configuration.